Developer-Guide.md

```diff
@@ -186,6 +186,13 @@ $ sudo docker run -ti --runtime kata-runtime busybox sh
 
 # Troubleshoot
 
+If you are unable to create a Kata Container, run the
+[`kata-collect-data.sh`](https://github.com/kata-containers/runtime/blob/master/data/kata-collect-data.sh.in)
+script (which is built from the
```

Comment:
Lines 191-192 suggested rewrite: script, which is built from the [runtime](https://github.com/kata-containers/runtime) repository,